File: LATIHAN4DPBO2023-main/C++/Course.cpp
// Saya Raisyad Jullfikar NIM 2106238 mengerjakan soal Latihan 4 dalam mata kuliah Desain Pemrograman Berorientasi 
// Objek untuk keberkahanNya maka saya tidak melakukan kecurangan seperti yang telah dispesifikasikan. Aamiin.

// Class course yang berisikan data course secara umum, seperti
// -> namamk

// Import Library
#include <iostream>
using namespace std;

class Course {
  // Deklarasi private attribute
  private:
    string __namaMK;

  // Constructor
  public:
    Course(string namaMK = "") {
      this->__namaMK = namaMK;
    }

  // Setter and Getter
  public:
    string get_namaMK() { // Method untuk mengambil data namaMK
      return this->__namaMK;
    }

    void set_namaMK(string namaMK) { // Method untuk menset atau mengubah atau menambah data namaMK
      this->__namaMK = namaMK;
    }

    ~Course() {} // Destructor
};

File: LATIHAN4DPBO2023-main/Python/Course.py
# Saya Raisyad Jullfikar NIM 2106238 mengerjakan soal Latihan 4 dalam mata kuliah Desain Pemrograman Berorientasi 
# Objek untuk keberkahanNya maka saya tidak melakukan kecurangan seperti yang telah dispesifikasikan. Aamiin.

# Class course yang berisikan data course secara umum, seperti
# -> namamk

class Course :
  # Deklar Private attr
  __namaMK = ""
  
  # CONSTRUCTOR
  # Constructor kosong yang mengisi valuenya dengan default value atau string kosong
  def __init__ (self, namaMK = "") :
    self.__namaMK = namaMK  
  
  # SETTER AND GETTER

  def get_namaMK(self) : # Method untuk mengambil data namaMK
    return self.__namaMK
  def set_namaMK(self, namaMK) : # Method untuk menset atau mengubah atau menambah data namaMK
    self.__namaMK = namaMK;